modules/programs/termite.nix

```diff
@@ -6,6 +6,14 @@ let
 
   cfg = config.programs.termite;
 
+  vteInitStr = ''
+    # https://github.com/thestinger/termite#id1
+    if [[ $TERM == xterm-termite ]]; then
+      . ${pkgs.gnome3.vte-ng}/etc/profile.d/vte.sh
+      __vte_prompt_command
```

Comment:
Looking at the link above it says `__vte_osc7` for the zsh case. Does it work with `__vte_prompt_command` as well?

Reply:
Looking at the vte.sh script again, it looks like we don't have to call any commands as the pre-prompt hook gets automatically installed:
https://gist.github.com/zimbatm/3f72cbd2d26e599e16b127a5e8799dd3#file-vte-sh-L61